README.md

```diff
@@ -1,2 +1,12 @@
 # MyL3-External-Data
 GSoC 2018 project
+
+Idea is to develop basic Infusion wrapper for My Life Long Learning Lab(MyL3). This wrapper will contain following data about learners imported from various external sources.
+
+* Import Weather data
+* Import Map/Location data
+* Import Google Fit data
+* Import FitBit data
+* Import Facebook Status
+
+More info about MyL3: [click here](https://wiki.fluidproject.org/display/fluid/%28Floe%29+Preference+Exploration+and+Self-Assessment)
```

Comment:
I would prefer a more descriptive link text than "click here". Maybe something like:

More information about the project may be found on the [MyL3 page of the Fluid wiki](https://wiki.fluidproject.org/display/fluid/%28Floe%29+Preference+Exploration+and+Self-Assessment).

Reply:
Updated